app/assets/javascripts/mapknitter/Map.js

```diff
@@ -88,24 +96,27 @@ MapKnitter.Map = MapKnitter.Class.extend({
 
           });
 
-          var corners = [ 
-                new L.latLng(warpable.nodes[0].lat,
-                             warpable.nodes[0].lon),
-                new L.latLng(warpable.nodes[1].lat,
-                             warpable.nodes[1].lon),
-                new L.latLng(warpable.nodes[3].lat,
-                             warpable.nodes[3].lon),
-                new L.latLng(warpable.nodes[2].lat,
-                             warpable.nodes[2].lon)
+          var corners = [
+            L.latLng(warpable.nodes[0].lat,
+              warpable.nodes[0].lon),
+            L.latLng(warpable.nodes[1].lat,
+              warpable.nodes[1].lon),
+            L.latLng(warpable.nodes[3].lat,
+              warpable.nodes[3].lon),
+            L.latLng(warpable.nodes[2].lat,
+              warpable.nodes[2].lon)
           ];
 
-          var img = new L.DistortableImageOverlay(
+          var img = L.distortableImageOverlay(
             warpable.srcmedium,
-            { 
+            {
               keymapper: false,
+              actions: mapknitter.imgActionArray(),
```

Comment:
this line is being called on mapknitter and the other time its called its called on `this`. but I believe they both work so it should just be called on `this`? Not sure because I was never getting the error you pointed out: 
```
Uncaught TypeError: this.imgActionArray is not a function
    at Object.<anonymous> (VM525 Map.self-c76b499773e2ed70a649cbce2df6162c1988da7e73aad1b2376e307b41ae80a3.js:106)
```
```suggestion
              actions: this.imgActionArray(),
```

Reply:
For images added at runtime by the user, `this` works, but for images that were loaded on the page already at load time, `this` was not being preserved through the nested constructors. So I assigned `this` to a local variable `mapknitter` with the correct scope, to get past the errors. In general, `this` usage is a bit brittle since it can so easily be overridden, so fixing it to a named locally scoped variable is a pretty good practice.